tests/DependencyInjection/PrintdealPandosearchExtensionTest.php

```diff
@@ -41,6 +41,33 @@ public function loadConfigurationProvider()
         ];
     }
 
+    /**
+     * @param $alias
+     * @dataProvider aliasesProvider
+     */
+    public function testAliasExist($alias)
+    {
+        $container = $this->createContainer();
+        $container->registerExtension(new CsaGuzzleExtension());
```

Comment:
yes, it works so, but in this case, for me, it is not clear why do you have 
```
$container->registerExtension(new CsaGuzzleExtension());
```
 at `testLoadConfiguration` but not at dataProvider